pkg/multicluster/cluster_metrics_management.go

```diff
@@ -62,13 +79,41 @@ func (cmm *ClusterMetricsMgr) Refresh() error {
 		if err != nil {
 			klog.Warningf("failed to request metrics api of cluster-(%s)", cluster.Name)
 		}
-		metrics := &ClusterMetrics{
+		cm := &ClusterMetrics{
 			IsConnected:         isConnected,
 			ClusterInfo:         clusterInfo,
 			ClusterUsageMetrics: clusterUsageMetrics,
 		}
-		m[cluster.Name] = metrics
+		m[cluster.Name] = cm
+		cluster.Metrics = cm
 	}
 	metricsMap = m
```

Comment:
Why not use a class variable?

Reply:
That means to make []VirtualCluster a variable of ClusterMetricsMgr?